.eslintignore

```diff
@@ -1 +1,10 @@
-**/*.html
\ No newline at end of file
+**/*.html
+
+_build
+_includes
+.github
+.vscode
+api
+docs
+lib
+node_modules
```

Comment:
I would put the generic ones first (node_modules, .github, etc.) and the project-specific ones after (possibly with a comment to that effect)